pom.xml

```diff
@@ -575,16 +474,6 @@
             </plugin>
             <plugin>
                <groupId>org.codehaus.mojo</groupId>
-               <artifactId>emma-maven-plugin</artifactId>
-               <version>${emma.plugin.version}</version>
```

Comment:
`emma` plugins should be for code coverage but these are pretty outdated and as far as I recall, we are now using `jacoco` for this